src/contoroller/account/account.js

```diff
@@ -59,12 +46,10 @@ class AccountController {
     return res.send("Deleted");
```

Comment:
```suggestion
    return res.status(204).end();
```

the status code 204 means that the deletion was successful, no need to send a string then.

Reply:
done